integration/common

```diff
@@ -80,6 +80,20 @@ launch_gethnet() {
   title "Geth is running."
 }
 
+launch_parity() {
+  printf -- "\033[34m%-50s\033[0m" "Checking for running ethereum node"
+  if curl -s $ETH_HTTP_URL >/dev/null; then
+    printf -- "[\033[32mrunning\033[0m]\n"
+    return
+  fi
+  printf -- "[\033[31mstopped\033[0m]\n"
+
+  title "Starting parity..."
+  $SRCROOT/internal/bin/devnet &>$SRCROOT/integration/devnet.log &
+  waitForResponse $ETH_HTTP_URL
```

Comment:
Wrong URL

Reply:
Can you suggest what the correct URL should be?